app/assets/stylesheets/layout_components/credit-card.scss

```diff
@@ -0,0 +1,34 @@
+.credit-card{
+  background-color: $cerulean;
+  border-radius: 3px;
+  padding: $base-padding;
+  max-width: 400px;
+  min-height: 216px;
+  position: relative;
+
+  &__logo-visa{
+    background: url(image-path("cc-type-visa-white.png")) no-repeat;
+    background-size: contain;
+    height: 50px;
+    width: 100px;
+    float:right;
+  }
+
+  &__logo-mc{
```

Comment:
With these 2 logo classes being nearly identical, would it make sense to use a mixin and pass the file name as a parameter?

```
@mixin card-logo($image-path) {
background: url(image-path($image-path)) no-repeat;
  background-size: contain;
  height: 50px;
  width: 100px;
  float: right;
}

&__logo-mc{
  @include card-logo("cc-type-mc-white.png");
}
```